easybuild/easyconfigs/c/CVXOPT/CVXOPT-1.3.1-foss-2022a.eb

```diff
@@ -32,8 +32,14 @@ use_pip = True
 sanity_pip_check = True
 download_dep_fail = True
 
-preinstallopts = 'CVXOPT_BUILD_FFTW=1 CVXOPT_BUILD_GSL=1 CVXOPT_BLAS_EXTRA_LINK_ARGS="$LIBLAPACK" '
-preinstallopts += 'CVXOPT_FFTW_EXTRA_LINK_ARGS="$LIBFFT" CVXOPT_SUITESPARSE_SRC_DIR=$EBROOTSUITESPARSE'
+preinstallopts = " ".join([
+    'CVXOPT_BUILD_FFTW=1',
+    'CVXOPT_BUILD_GSL=1',
+    'CVXOPT_BLAS_EXTRA_LINK_ARGS="$LIBLAPACK"',
```

Comment:
Does CVXOPT need Lapack or just BLAS?
In either case using just LIBLAPACK is wrong.
If it need both then it needs $LIBLAPACK $LIBBLAS (they may not be the same)
and if it just needs BLAS then it should be just $LIBBLAS

Reply:
It seems that both BLAS and LAPACK are required. I fix the flags now.